src/google/protobuf/descriptor.cc

```diff
@@ -7722,27 +7735,22 @@ bool DescriptorBuilder::OptionInterpreter::SetOptionValue(
                  uninterpreted_option_->negative_int_value(),
                  option_field->type(), unknown_fields);
       } else {
-        return AddValueError("Value must be integer for int64 option \"" +
-                             option_field->full_name() + "\".");
+        return AddValueError(ValueMustBeInt<int64>("int64", option_field->full_name()));
       }
```

Comment:
please use the `_t` variant of all the types when naming the C++ type (the error message should still refer to the proto type as you do now)

Reply:
Done